src/ContainerTerminal.jsx

```diff
@@ -95,7 +99,7 @@ class ContainerTerminal extends React.Component {
         var realWidth = this.state.term._core._renderService.dimensions.actualCellWidth;
         var cols = Math.floor((width - padding) / realWidth);
         this.state.term.resize(cols, 24);
-        client.resizeContainersTTY(this.props.system, this.state.container, cols, 24)
+        client.resizeContainersTTY(this.props.system, this.state.sessionId, this.props.tty, cols, 24)
```

Comment:
Why don't you keep using the existing state 'container' here? It has the same value. (Or does it change somewhere I don't see?)

Reply:
It does, when we connect through exec, it is not container ID, but exec ID. It is set up [here](https://github.com/cockpit-project/cockpit-podman/pull/461/files#diff-ce6eac7021a57afa8c484caf011f3e15R195)

Reply:
Ack thanks, thought that's still the containerId but aparently it's different value of that (r.Id)